### PR TITLE
Display job posting attributes & flags

### PR DESCRIPTION
**Pull request by lukasbischof**
<video src="https://user-images.githubusercontent.com/8350985/193429188-33deb5e3-df89-4be2-a921-61d683c07613.mp4">

### CODE REVIEW

**Review by TheFehr — COMMENTED** [review]
Is there one stat cut of at 0:15

**Review by TheFehr — APPROVED** [review]
Butiful new feature, just like the map, widget and login feature will be one day

File: app/javascript/stylesheets/home/_jobs.scss [hunk 1]
```diff
@@ -155,3 +155,39 @@ section.job {
     }
   }
 }
+
+.job-posting-attributes-section {
+  display: grid;
```
Comment (lukasbischof) [inline app/javascript/stylesheets/home/_jobs.scss:160]: 🔲🔲🔲
🔲🔲🔲
🔲🔲🔲

**Comment by lukasbischof** [comment]
well at least that's a mergeable feature for once in a while

**Comment by lukasbischof** [comment]
And not like an infinitely long wip

**Pull request merged**